conf.yaml

```diff
@@ -109,6 +109,19 @@ variables:
 
 toc_extra: extra/docs_landing.html
 contents:
+    -   title:      "Welcome to Elastic"
+        prefix:     en/welcome-to-elastic
+        current:    main
+        index:      welcome-to-elastic/index.asciidoc
+        branches:   [ {main: master} ]
+        chunk:      1
+        tags:       Elastic/Welcome
+        subject:    Welcome to Elastic
+        sources: 
+          -
+            repo:   tech-content
```

Comment:
You'll need to add the `tech-content` repo to the top of this file (under `repos`) in order for this to work.

Reply:
Thanks Greg, fixed that up.